docs/install.md

```diff
@@ -43,10 +43,10 @@ sudo apt install python3.8 -y
 
 Since LISA is installed from source, `git` is necessary. And some packages need to be built, so the build tools are also needed.
 
-In Linux, it needs `git` and `gcc`. Below is depended packages on Ubuntu.
+In Linux, it needs `git`, `gcc`, and other Azure dependencies. Below is depended packages on Ubuntu.
 
 ```bash
-sudo apt install git gcc
+sudo apt install git gcc libgirepository1.0-dev libcairo2-dev gir1.2-secret-1
```

Comment:
Wait, I thought cairo et al were handled by Poetry now, what's going on?

Reply:
Oh, these are the -dev ones, true. What's up with that gir thing? We don't have it on the CI and I don't recall having seen errors without it.

Reply:
removed gir1.2-secret-1. I got it from WSL, as it's not supported, so I will remove it.